bluesky/tests/test_callbacks.py

```diff
@@ -52,14 +47,18 @@ def cb(name, doc):
 
     # by default (with ignore... = True) it warns
     with pytest.warns(UserWarning):
-        RE(stepscan(det, motor), cb)
+        RE(stepscan(hw.det, hw.motor), cb)
 
     RE.ignore_callback_exceptions = False
     with pytest.raises(Exception):
-        RE(stepscan(det, motor), {'all': cb})
+        RE(stepscan(hw.det, hw.motor), cb)
 
+    RE.ignore_callback_exceptions = False
```

Comment:
duplicate code?

Reply:
intentional future-proofing against changed to one sub-section of the test or another

Reply:
:sheep: